ldk/javascript/src/whisper/index.ts

```diff
@@ -264,6 +284,33 @@ export interface WhisperAptitude {
   create(whisper: NewWhisper): Promise<Whisper>;
 }
 
-export function create(whisper: NewWhisper): Promise<Whisper> {
-  return promisifyWithParam(whisper, oliveHelps.whisper.create);
+function addUpdateHandler(whisper: OliveHelps.Whisper): Whisper {
```

Comment:
Will it be cleaner to have a whole 'addUpdateHandler' function to be moved to the mapper? Basically you doing a 'mapToWhisper(whisper: OliveHelps.Whisper)'